File: day02/Nilesh_18311/TShirt.java
package com.sonata;

public class TShirt {
	String color;
	String material;
	String design;
	
	 TShirt(String color,String material,String design)
	{
		this.color=color;
		this.material=material;
		this.design=design;
	}
	TShirt()
	{
		
	}
	TShirt(String exLa,String mate)
	{
		
	}
	public void display()
	{
		System.out.println("Tshirt is having color "+color+" and made up of "+material+" design "+design);
	}

	public static void main(String[] args) {
			TShirt small= new TShirt("Red","Cotton","Polo");
			TShirt medium=new TShirt("Green", "polyster", "Round Neck");
			TShirt ex_large=new TShirt("Black", "Cotton", "Polo");
			small.display();
			medium.display();
			ex_large.display();
	}

}

File: JS/Day1Assign/T_shirt.ts
class T_shirt{
    color:string;
    material:string;
    design:string;

    constructor(color:string,material:string,design:string){
        this.color=color;
        this.material = material;
        this.design=design;
    }
    
    x_large(material:string):string{
        this.material=material;
        return this.material;
    }
     display():void{
        console.log("Color: "+this.color+", material :"+this.material+",design: " + this.design);
    }

}
let small=new T_shirt("Red","1 meter",'Round Neck');
small.display();
let large= new T_shirt("Green","1 meter",'Round Neck');
large.display();
let xt_large= new T_shirt("Yellow","1_meter","Polo");
xt_large.material="2 meters";
xt_large.display();